### DISCUSSION TITLE
郭宇老师理解 Plonk（五）：多项式承诺 -线性化部分

### DISCUSSION THREAD

**Discussion by miles-six** [zkp-co-learning/zkp-co-learn#97] [Q&A（提问题在隔壁~）]
通过v折叠4个多项式
<img width="1007" alt="image" src="https://github.com/Antalpha-Labs/zkp-co-learn/assets/66052478/06dc79c1-3329-4b88-a9b0-ea28b84baf1f">

为什么不是下图形式
<img width="680" alt="image" src="https://github.com/Antalpha-Labs/zkp-co-learn/assets/66052478/a2a7afb3-314d-4cd3-a349-b472935a6e22">
这样构造的原因是什么

**Comment by readygo67** [comment]
在原来的形式中，当 $X=\zeta$ 时，有

$L(x) = 0$ 

$\Big(L(X) + \nu\cdot (f_1(X)-\bar{f}_1)+\nu^2\cdot (h_1(X)-\bar{h}_1)+\nu^3\cdot (h_2(X)-\bar{h}_2)\Big) = 0$

但是在
![image](https://github.com/Antalpha-Labs/zkp-co-learn/assets/78890754/9f5a19b7-4c6b-44f6-b183-6624052b9432)
中， $\Big(L(X) + \nu\cdot (f_1(X))+\nu^2\cdot (h_1(X))+\nu^3\cdot (h_2(X))\Big)$  是不一定被 $x- \zeta$ 整除的， 因为f1(x), h1(x), h2(x) 不 一定能被 $x- \zeta$ 整除。

我的理解，
1.  $q(X)$ 利用 $\nu$ 折叠 $(L(X), f_1(X),h_1(X),h_2(X))$ 这四个承诺。
2. 利用s-z 引理隐含的证明了 $\bar{f_1}=f_1(\zeta)$， $\bar{h}_1=h_1(\zeta)$， $\bar{h}_2=h_2(\zeta)$
